Testing/SystemTests/tests/framework/D7AbsoluteCrossSectionsTest.py

```diff
@@ -0,0 +1,257 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import systemtesting
+from mantid.simpleapi import *
+
+
+class ILL_D7_Powder_Test(systemtesting.MantidSystemTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls._original_facility = config['default.facility']
+        cls._original_instrument = config['default.instrument']
+        cls._data_search_dirs = config.getDataSearchDirs()
+
+    @classmethod
+    def tearDownClass(cls):
+        config['default.facility'] = cls._original_facility
+        config['default.instrument'] = cls._original_instrument
+        config.setDataSearchDirs(cls._data_search_dirs)
+        mtd.clear()
+
+    def setUp(self):
```

Comment:
I think you forgot to remove those.

Reply:
This should go to the setupClass as well. Even though after all there is only one test in the suite, so it doesn't really matter, but just to get the habit in case other tests will be added to the same suite.

Reply:
That was my initial approach and in that case, the test fails due to not being able to find data. The settings are therefore not properly set for the test, e.g. the chosen instrument remains empty. Using `setUp` explicitly sets the config in a way that is visible by the test.

Reply:
strange, are the setUpClass and tearDownClass run at all? Perhaps they are only for unit tests, and not system tests, can you check the base class of systemtest if they exist?

Reply:
MantidSystemTest actually does not have `setUpClass` nor `tearDownClass` methods, only `setUp` and both 'tearDown' and `cleanup` exist there.

Reply:
OK, please check if they are run, if not, then they are only for unit tests, so revert those changes.

Reply:
I believe they are not run at all in the state as implemented in 8bd7b9b. I am effectively reverting this change with 7e407fb commit.

Reply:
But they are still marked as class methods and the argument is cls.